plugins/kotlin-dataframe/testData/box/groupBy_sum.kt

```diff
@@ -0,0 +1,54 @@
+import org.jetbrains.kotlinx.dataframe.*
+import org.jetbrains.kotlinx.dataframe.annotations.*
+import org.jetbrains.kotlinx.dataframe.api.*
+import org.jetbrains.kotlinx.dataframe.io.*
+
+fun box(): String {
+    // multiple columns
+    val personsDf = dataFrameOf("name", "age", "city", "weight", "height", "yearsToRetirement")(
+        "Alice", 15, "London", 99.5, "1.85", 50,
+        "Bob", 20, "Paris", 140.0, "1.35", 45,
+        "Charlie", 100, "Dubai", 75, "1.95", 0,
+        "Rose", 1, "Moscow", 45.3, "0.79", 64,
+        "Dylan", 35, "London", 23.4, "1.83", 30,
+        "Eve", 40, "Paris", 56.7, "1.85", 25,
+        "Frank", 55, "Dubai", 78.9, "1.35", 10,
+        "Grace", 29, "Moscow", 67.8, "1.65", 36,
+        "Hank", 60, "Paris", 80.2, "1.75", 5,
+        "Isla", 22, "London", 75.1, "1.85", 43,
+    )
+
+    // scenario #0: all numerical columns
+    val res0 = personsDf.groupBy { city }.sum()
+    val sum01: Int? = res0.age[0]
+    // TODO: Compilation error - actual type it(kotlin.Number & kotlin.Comparable<*>)
+    // `val sum02: Double? = res0.weight[0]
+
+    // scenario #1: particular column
+    val res1 = personsDf.groupBy { city }.sumFor { age }
```

Comment:
sumFor, maxFor need support too because now "age" column comes from T
```
public fun <T, C : Number> Grouped<T>.sumFor(columns: ColumnsForAggregateSelector<T, C?>): DataFrame<T> =
    Aggregators.sum.aggregateFor(this, columns)
```
While there's `age`, problem is that there's also column `height` in compile time schema, but there's no such column in runtime. In tests i have this utility to compare if compile time schema is compatible with runtime:
"res1.compareSchemas()"